apps/crn-frontend/src/network/teams/api.ts

```diff
@@ -576,44 +576,15 @@ export const uploadManuscriptFileViaPresignedUrl = async (
   }
 };
 
-export const getPresignedUrl = async (
-  filename: string,
+export const downloadFullComplianceDataset = async (
   authorization: string,
-  contentType?: string,
-  action: FileAction = 'upload',
-): Promise<{ presignedUrl: string }> => {
-  const resp = await fetch(`${API_BASE_URL}/files/get-url`, {
-    method: 'POST',
-    headers: {
-      authorization,
-      'Content-Type': 'application/json',
-      ...createSentryHeaders(),
-    },
-    body: JSON.stringify({ filename, action, contentType }),
-  });
-
-  let response;
-  try {
-    response = await resp.json();
-  } catch (error) {
-    throw new BackendError(
-      `Failed to parse JSON response when generating presigned URL. Status: ${resp.status}`,
-      {
-        error: 'ParseError',
-        message: 'Failed to parse JSON response',
-        statusCode: resp.status,
-      },
-      resp.status,
-    );
-  }
-
-  if (!resp.ok) {
-    throw new BackendError(
-      `Failed to generate presigned URL. Expected status 200. Received status ${resp.status}: ${resp.statusText}`,
-      response,
-      resp.status,
-    );
-  }
+): Promise<string> => {
+  const { presignedUrl: downloadUrl } = await getPresignedUrl(
+    'ComplianceFullDataset.csv',
```

Comment:
I think we can send the filename without the `.csv`. When we download the file, it's coming with a `.csv` in the middle and in the end

<img width="1226" height="924" alt="Image" src="https://github.com/user-attachments/assets/3d2b5f24-bf17-4d06-bac6-764a0055ebdf" />

```suggestion
    'ComplianceFullDataset',
```